.gitlab-ci.yml

```diff
@@ -7,9 +7,13 @@ include:
   file: private.yml
 
 cache:
-  key: ${CI_COMMIT_REF_SLUG}
+  key:
+    files:
+      - package.json
```

Comment:
Should we also check for changes in  `package-lock.json`? E.g. security updates could only update `package-lock.json`.

Reply:
`package.json` is fine for now, we can add package-lock.json when the issue arises.